downloader/download_manager.py

```diff
@@ -0,0 +1,25 @@
+from downloader.history_downloader.history_manager import HistoryDownloader
+from downloader.history_saver.save_manager import HistorySaver
+
+
+class DownloadManager:
+
+    def __init__(self, exchange_name: str, crypto_symbol: str, days_back: int):
+        self.crypto_symbol = crypto_symbol
+        self.days_back = days_back
+        self.exchange_name = exchange_name
+
+    def download_history(self):
+        cryptocurrency_data_downloader = HistoryDownloader(exchange_name=self.exchange_name,
+                                                           crypto_symbol=self.crypto_symbol,
+                                                           days_back=self.days_back)
+
+        cryptocurrency_data_history = cryptocurrency_data_downloader.get_history()
+
+        # here will be db saving method
+        HistorySaver(cryptocurrency_data_history).save_history()
+
+
+# example of downloading data from binance exchange about bitcoin from last 4 days

```

Comment:
Examples should be put in docstrings for methods eg.:

```python
def example_generator(n):
    """Generators have a ``Yields`` section instead of a ``Returns`` section.

    Args:
        n (int): The upper limit of the range to generate, from 0 to `n` - 1.

    Yields:
        int: The next number in the range of 0 to `n` - 1.

    Examples:
        Examples should be written in doctest format, and should illustrate how
        to use the function.

        >>> print([i for i in example_generator(4)])
        [0, 1, 2, 3]

    """
    for i in range(n):
        yield i
```

Reply:
Alright, it should be fine now